plugins/Goals/lang/en.json

```diff
@@ -38,7 +38,7 @@
         "Conversions": "%s conversions",
         "ConversionsOverview": "Conversions Overview",
         "ConversionsOverviewBy": "Conversions overview by type of visit",
-        "CreateNewGOal": "Create a new Goal",
+        "CreateNewGOal": "Add a new Goal",
```

Comment:
there is already a string `Goal_AddNewGoal` so let's reuse this one and delete `Goal_CreateNewGOal`